### PR TITLE
Add a example.

### PR DESCRIPTION
**Pull request by maobaolong**

### CODE REVIEW

**Comment by maobaolong** [comment]
@NoXF  I wrote an example to show how to use this useful lib, PTAL.

File: example/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
+extern crate futures;
+extern crate tokio_core;
+
+use std::str;
+
+use oss_rust_sdk::prelude::*;
+use std::collections::HashMap;
+use reqwest::{r#async as async_reqwest, Error};
```
Comment (NoXF) [inline example/src/main.rs]: it seems not to need import reqwest

File: example/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+use reqwest::header::{HeaderMap, DATE};
+use futures::{Future, Stream};
```
Comment (NoXF) [inline example/src/main.rs]: not need Stream too

File: example/src/main.rs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+use tokio_core::reactor::Core;
+
+fn main() {
+    println!("Hello, world!");
+    syncGetObject();
```
Comment (NoXF) [inline example/src/main.rs]: snake_case is generally used in Rust

File: example/src/main.rs [hunk 4]
```diff
@@ -0,0 +1,42 @@
+
+extern crate futures;
+extern crate tokio_core;
```
Comment (NoXF) [inline example/src/main.rs]: "extern crate" can implicit by edition 2018

File: example/src/main.rs (hunk 1 above)
Comment (maobaolong) [reply example/src/main.rs]: done. Only Error is used here in reqwest.

File: example/src/main.rs (hunk 2 above)
Comment (maobaolong) [reply example/src/main.rs]: Yeah, remove it.

File: example/src/main.rs (hunk 4 above)
Comment (maobaolong) [reply example/src/main.rs]: OK, remove it all.

**Comment by maobaolong** [comment]
@NoXF  Thank you for your review, it getting more clearly now, thank you for improve my code.

File: example/src/main.rs (hunk 3 above)
Comment (maobaolong) [reply example/src/main.rs]: Done

**Pull request merged**